content/api/resolve.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Resolve
```

Comment:
What is the difference with this and the [configuration/resolve](https://github.com/webpack/webpack.js.org/blob/develop/content/configuration/resolve.md) page?

I see that this has some more options documented, so we can merge this page into the configuration one.